multi-agent/lib/mas/session/management/user_session_manager.py

```
@@ -54,6 +54,14 @@ def get_blueprint_metadata(self, blueprint_id: str) -> Dict[str, Any]:
         except KeyError:
             return {}
 
+    def get_blueprint_name(self, blueprint_id: str) -> str:
+        """Get blueprint display name, falling back to blueprint_id if not found."""
+        try:
+            doc = self._bp_service.get_blueprint_draft_doc(blueprint_id)
+            return doc.spec_dict.get("name", blueprint_id)
+        except (KeyError, Exception):
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)